CadRevealFbxProvider/BatchUtils/ScaffoldPartOptimizers/ScaffoldPartOptimizer.cs

```diff
@@ -0,0 +1,9 @@
+namespace CadRevealFbxProvider.BatchUtils.ScaffoldPartOptimizers;
+using CadRevealComposer.Tessellation;
+
+public abstract class ScaffoldPartOptimizer
+{
+    public abstract string GetName();
+    public abstract Mesh[] Optimize(Mesh mesh);
+    public abstract string[] GetPartNameTriggerKeywords();
+}
```

Comment:
You could use an interface instead here, not sure if it makes a difference though